### ISSUE TITLE
Connection to a remote SSH server w/ Password Auth error

### ISSUE THREAD

**Issue by aIIclear** [calebstewart/pwncat#47]
**Describe the bug**
error connect via ssh, `pwncat -s -H <IP> -u user --password "passwd"`

**Describe the target system**
Linux parrot 5.7.0-2parrot2-amd64 SMP Sebian 5.7.10-1parrot2 (2020-07-31)

**Screenshots**

![Screenshot 2020-08-11 at 12 37 28](https://user-images.githubusercontent.com/46959099/89883139-6e64c600-dbd0-11ea-9b60-c09c038011b3.png)

**Comment by petofikid** [comment]
`└──╼ $pwncat -s -H 10.10.13.125 -u jan --password "armando"
Traceback (most recent call last):
  File "/home/allclear/Desktop/pwncat/pwncat-env/bin/pwncat", line 11, in <module>
    load_entry_point('pwncat==0.3.1', 'console_scripts', 'pwncat')()
  File "/home/allclear/Desktop/pwncat/pwncat-env/lib/python3.8/site-packages/pwncat-0.3.1-py3.8.egg/pwncat/__main__.py", line 28, in main
    pwncat.victim.command_parser.dispatch_line(
  File "/home/allclear/Desktop/pwncat/pwncat-env/lib/python3.8/site-packages/pwncat-0.3.1-py3.8.egg/pwncat/commands/__init__.py", line 290, in dispatch_line
    command.run(args)
  File "/home/allclear/Desktop/pwncat/pwncat-env/lib/python3.8/site-packages/pwncat-0.3.1-py3.8.egg/pwncat/commands/connect.py", line 229, in run
    pwncat.victim.connect(chan)
  File "/home/allclear/Desktop/pwncat/pwncat-env/lib/python3.8/site-packages/pwncat-0.3.1-py3.8.egg/pwncat/remote/victim.py", line 279, in connect
    hostname_path = self.run("which hostname").strip().decode("utf-8")
  File "/home/allclear/Desktop/pwncat/pwncat-env/lib/python3.8/site-packages/pwncat-0.3.1-py3.8.egg/pwncat/remote/victim.py", line 1035, in run
    sdelim, edelim = self.process(cmd, delim=wait, timeout=timeout)
  File "/home/allclear/Desktop/pwncat/pwncat-env/lib/python3.8/site-packages/pwncat-0.3.1-py3.8.egg/pwncat/remote/victim.py", line 1089, in process
    data = self.client.recv(len(command), socket.MSG_PEEK)
TypeError: recv() takes 2 positional arguments but 3 were given`

**Comment by juushya** [comment]
@petofikid refer to [#37](https://github.com/calebstewart/pwncat/issues/37#issuecomment-660720173). This bug will be fixed once once this PR #1697 (https://github.com/paramiko/paramiko/pull/1697) is merged. Or if you want it right away, you can always manually pick up the updated files from [here](https://github.com/calebstewart/paramiko/commit/5db6415b04b02ed743bf198d531f508415f95dee) and put them in your local pwncat/paramiko repo.

Here're the two (2) files that will be updated with the PR:
```
/opt/pwncat/pwncat-env/lib/python3.8/site-packages/paramiko-2.7.1-py3.8.egg/paramiko/buffered_pipe.py
/opt/pwncat/pwncat-env/lib/python3.8/site-packages/paramiko-2.7.1-py3.8.egg/paramiko/channel.py
```

```
(pwncat-env) root@kali:/opt/pwncat# pwncat -s -H localhost --user "user" --password "user" 
[16:33:20] new host w/ hash d80b2895957bcc4b58da66e817b871c2.    victim.py:329
           pwncat running in /usr/bin/bash                       victim.py:363
           pwncat is ready 🐈                                     victim.py:762

(remote) user@kali:/$ 
```

**Comment by petofikid** [comment]
> @petofikid refer to [#37](https://github.com/calebstewart/pwncat/issues/37#issuecomment-660720173). This bug will be fixed once once this PR #1697 (https://github.com/paramiko/paramiko/pull/1697) is merged. Or if you want it right away, you can always manually pick up the updated files from [here](https://github.com/calebstewart/paramiko/commit/5db6415b04b02ed743bf198d531f508415f95dee) and put them in your local pwncat/paramiko repo.
> 
> 
> 
> Here're the two (2) files that will be updated with the PR:
> 
> ```
> 
> /opt/pwncat/pwncat-env/lib/python3.8/site-packages/paramiko-2.7.1-py3.8.egg/paramiko/buffered_pipe.py
> 
> /opt/pwncat/pwncat-env/lib/python3.8/site-packages/paramiko-2.7.1-py3.8.egg/paramiko/channel.py
> 
> ```
> 
> 
> 
> ```
> 
> (pwncat-env) root@kali:/opt/pwncat# pwncat -s -H localhost --user "user" --password "user" 
> 
> [16:33:20] new host w/ hash d80b2895957bcc4b58da66e817b871c2.    victim.py:329
> 
>            pwncat running in /usr/bin/bash                       victim.py:363
> 
>            pwncat is ready 🐈                                     victim.py:762
> 
> 
> 
> (remote) user@kali:/$ 
> 
> ```

thanks you my friend, i just got it work!
